skymp5-server/cpp/server_guest_lib/MpObjectReference.h

```diff
@@ -124,6 +124,7 @@ class MpObjectReference
   void SetInventory(const Inventory& inv);
   void AddItem(uint32_t baseId, uint32_t count);
   void AddItems(const std::vector<Inventory::Entry>& entries);
+  void ForceEquip(uint32_t baseId);
```

Comment:
please remove this method and replace with sending a packet to a client to equip something